test/suites/dev/test-author/test-author-failed-association.ts

```diff
@@ -36,8 +36,8 @@ describeSuite({
         ).partialFee.toBigInt();
 
         const { result } = await context.createBlock(
-          api.tx.authorMapping.addAssociation(ALITH_SESSION_ADDRESS).signAsync(baltathar),
-          { allowFailures: true }
+          api.tx.authorMapping.addAssociation(ALITH_SESSION_ADDRESS),
+          { allowFailures: true, signer: baltathar }
```

Comment:
I understand having the possibility to provide the global signer for all tx is good, but in this specific case, any reason to not keep signAsync ?

Reply:
No reason, i did it differently here i think to test out the new signer syntax. Have changed back for consistency